cafebazaar/BazaarPay
# Feature autoLogin

AutoLogin added; from this version, you can use SDK without login in SDK by paymentURL

### **test scenarios**

- Check with different paymentURL (autologin/ without autoLogin)
- Check all API with autoLogin in SDK (they should have `checkOutToken` in the body of request or `query params`)
- Check logOut flow after autoLogin
- Check increase balance with AutoLoign
- Check sample app
